deploy/catalog_resources/redhat/activemq-artemis-operator.v0.4.0.clusterserviceversion.yaml

```diff
@@ -187,7 +205,7 @@ spec:
                 containers:
                   - name: activemq-artemis-operator
                     # Replace this with the built image name
-                    image: activemq-artemis-operator:latest
+                    image: quay.io/rh_messaging/activemq-artemis-operator:0.4.0
                     command:
                       - /usr/local/bin/entrypoint
```

Comment:
@redhatHameed Was the referenced image built against the tag where this was the correct location for the entrypoint?

Reply:
update the image and now using the entry point  '/home/activemq-artemis-operator/bin/entrypoint' as same current Dockerfile version.